compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -4510,7 +4519,8 @@ private ParserRuleContext getNextRuleForTypeDescriptor() {
                 return ParserRuleContext.CLOSE_PARENTHESIS;
             case TYPE_DESC_IN_TUPLE:
             case STMT_START_BRACKETED_LIST:
-                return ParserRuleContext.TYPE_DESC_IN_TUPLE_RHS;
+            case TUPLE_MEMBERS:
```

Comment:
We should not reach here for the `TUPLE_MEMBERS`. We need to end the context for `TYPE_DESC_IN_TUPLE`, before we go into `TYPE_DESC_IN_TUPLE_RHS`. 
```java
            case TYPE_DESC_IN_TUPLE:
                endContext();
                return ParserRuleContext.TYPE_DESC_IN_TUPLE_RHS;
            case STMT_START_BRACKETED_LIST:
                return ParserRuleContext.TYPE_DESC_IN_TUPLE_RHS;
```